### PR TITLE
BAU: Tidy up code

### PR DESCRIPTION
**Pull request by adityapahuja**
Update to use latest version of localstack.
Remove unused queue url result variable.
Move non unit test files to utils.
Add a builder to build a test event message to remove duplicate code.

Authors: @adityapahuja

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-before_install:
-# this is a hack, so PR builds can be tested in travis
-  - perl -i -0pe 's/maven[\s\{]+[^\{\}]*\}/maven { url \"https:\/\/build.shibboleth.net\/nexus\/content\/groups\/public\" \n url \"https:\/\/repo1.maven.org\/maven2\" \n jcenter() }/gms' build.gradle
\ No newline at end of file
+services:
+  - docker
+before_script:
+  - docker build -f PublicDockerfile -t verify-event-emitter .
+script:
+  - docker run --env VERIFY_USE_PUBLIC_BINARIES verify-event-emitter /bin/sh -c "./gradlew clean build"
```
[the remark this answers is not in the document]
Comment (adityapahuja) [reply .travis.yml]: `--env` allows Docker to fetch the initialised environment variable VERIFY_USE_PUBLIC_BINARIES for gradle since gradle is going to use the environment variable to choose whitelisted artifactory or bintray to download Java libraries for event emitter.
